### PR TITLE
Newest Even Newer New Crystal Hollow Items, this time with a new amazing weird pet that needs to be forged for way too long

### PR DESCRIPTION
**Pull request by Strebbypatty**
This will prob be the last new stuff for crystal hollows sadly, so no more epic titles of it after this
DONT MERGE YET IRONM00N

### CODE REVIEW

File: items/AMMONITE;0.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
...rties:{textures:[0:{Value:\"ewogICJ0aW1lc3RhbXAiIDogMTYyMDExNTI5NDE1NiwKICAicHJvZmlsZUlkIiA6ICI0NWY3YTJlNjE3ODE0YjJjODAwODM5MmRmN2IzNWY0ZiIsCiAgInByb2ZpbGVOYW1lIiA6ICJfSnVzdERvSXQiLAogICJzaWduYXR1cmVSZXF1aXJlZCIgOiB0cnVlLAogICJ0ZXh0dXJlcyIgOiB7CiAgICAiU0tJTiIgOiB7CiAgICAgICJ1cmwiIDogImh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvYTA3NGE3YmQ5NzZmZTZhYmExNjI0MTYxNzkzYmU1NDdkNTRjODM1Y2Y0MjIyNDNhODUxYmEwOWQxZTY1MDU1MyIsCiAgICAgICJtZXRhZGF0YSIgOiB7CiAgICAgICAgIm1vZGVsIiA6ICJzbGltIgogICAgICB9CiAgICB9CiAgfQp9\"}]}},display:{Lore:[0:\"§8Fishing Pet\",1:\"\",2:\"§7Sea Creature Chance: §a0%\",3:\"\",4:\"§6Heart of the Sea\",5:\"§7§7Each Heart of the Mountain\",6:\"§7level grants §3+0α Sea Creature\",7:\"§3Chance.\",8:\"\",9:\"§7§eRight-click to add this pet to\",10:\"§eyour pet menu!\",11:\"\",12:\"§f§lCOMMON\",13:\"\",14:\"§eItems Required\",15:\"§6Helix §7x1\",16:\"§6300,000 Coins\",17:\"\",18:\"§7Duration: §b10d 4h 48m!\"],Name:\"§7[Lvl 1] §fAmmonite\"},ExtraAttributes:{petInfo:\"{\\\"type\\\":\\\"AMMONITE\\\",\\\"active\\\":false,\\\"exp\\\":0.0,\\\"tier\\\":\\\"COMMON\\\",\\\"hideInfo\\\":false,\\\"candyUsed\\\":0}\",originTag:\"PET_MENU\",id:\"AMMONITE;0\",AttributeModifiers:[]}",
```
Comment (jani270) [inline items/AMMONITE;0.json:4]: You forgot to update NBT Data here.

File: items/GEMSTONE_CHAMBER.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+  "itemid": "minecraft:skull",
+  "displayname": "§5Gemstone Chamber",
+  "nbttag": "{overrideMeta:1b,HideFlags:254,SkullOwner:{Id:\"558648c0-4dda-371c-87d8-f983a7bc58ca\",Properties:{textures:[0:{Value:\"eyJ0ZXh0dXJlcyI6eyJTS0lOIjp7InVybCI6Imh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvZmI4MDk4ODlkYTM2YWRmNTYxODU4MWJiZjdiNjZkMmQ4ODM5ZTJlYjcyNTRjMzMzMmU0ZjNhMjMwZmEifX19\"}]}},display:{Lore:[0:\"§7Can be applied in Anvils to some\",1:\"§7items to add §dGemstone §7slots,\",2:\"§7which can then be filled at the\",3:\"§7§dGemstone Grinder§7.\",4:\"\",5:\"§5§lEPIC\",6:\"\",7:\"§eItems Required\",8:\"§aWorm Membrane §7x100\",9:\"§9Gemstone Mixture §7x1\",10:\"§625,000 Coins\",11:\"\",12:\"§7Duration: §b3h 24m\"],Name:\"§5Gemstone Chamber\"},ExtraAttributes:{id:\"GEMSTONE_CHAMBER\"},AttributeModifiers:[]}",
```
Comment (jani270) [inline items/GEMSTONE_CHAMBER.json:4]: You forgot to update NBT Data here. v2

**Comment by Strebbypatty** [comment]
The pet takes 12 days to forge as you may see so don't merge yet please mr. IRONM00N, I will add all rarities and stats hopefully once thats done.

**Review by jani270 — CHANGES_REQUESTED** [review]
Me when not updated nbt data

**Comment by Strebbypatty** [comment]
> Me when not updated nbt data

My brain is too big

**Comment by DeDiamondPro** [comment]
Bald

**Review by jani270 — APPROVED** [review]
👍

**Review by DeDiamondPro — APPROVED** [review]
epic

**Review by IRONM00N — APPROVED** [review]
lgtm

**Pull request merged**